### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by synicalsyntax**
#55

### CODE REVIEW

**Comment by zulipbot-test** [comment]
Hello @zulipbot-testing/server-development members, this pull request references an issue with the "area: documentation (developer)" label, so you may want to check it out!

File: README.md [hunk 1]
```diff
@@ -1 +1,2 @@
-# test
\ No newline at end of file
+# testbug
```
Comment (synicalsyntax) [inline README.md]: No

File: README.md [hunk 2]
```diff
@@ -1 +1,2 @@
-# test
```
Comment (zulipbot-test) [inline README.md]: Single comments don't count

File: README.md (hunk 1 above)
Comment (zulipbot-test) [inline README.md]: No

**Comment by zulipbot-test** [comment]
No

File: README.md (hunk 1 above)
Comment (zulipbot-test) [inline README.md]: don't assign

File: README.md (hunk 2 above)
Comment (zulipbot-test) [inline README.md]: really

File: README.md [hunk 3]
```diff
@@ -1 +1,3 @@
...
\ No newline at end of file
+# testbug
+d
+fd
```
Comment (zulipbot-test) [inline README.md]: I said don't assign smh

File: README.md (hunk 2 above)
Comment (zulipbot-test) [inline README.md]: But here

File: README.md (hunk 1 above)
Comment (zulipbot-test) [inline README.md]: what about ehre

File: README.md [hunk 4]
```diff
@@ -1 +1,3 @@
-# test
```
Comment (zulipbot-test) [inline README.md]: TTHIS

Comment (zulipbot-test) [inline README.md]: One

**Comment by zulipbot-test** [comment]
@zulipbot-test claim

**Pull request closed without merging**